apps/root/src/pages/strategy-guardian-detail/strategy-management/withdraw/hooks/useEarnWithdrawActions.tsx

```diff
@@ -179,15 +179,34 @@ const useEarnWithdrawActions = ({ strategy }: UseEarnWithdrawActionsParams) => {
 
       try {
         setModalLoading({
-          content: (
-            <Typography variant="bodyRegular">
-              <FormattedMessage
-                description="earn.strategy-management.withdraw.modal.loading"
-                defaultMessage="You are now withdrawing funds from {farm}. Time to enjoy the rewards you've cultivated 🎉"
-                values={{ farm: strategy.farm.name }}
-              />
-            </Typography>
-          ),
+          content:
+            assetWithdrawType === WithdrawType.DELAYED ? (
+              <Typography variant="bodyRegular">
+                <FormattedMessage
+                  description="earn.strategy-management.delayed-withdraw.modal.loading"
+                  defaultMessage="Your funds are being queued for withdrawal from {farm}. They'll be ready for you soon! 🕒. {rewards}"
+                  values={{
+                    farm: strategy.farm.name,
+                    rewards: withdrawRewards ? (
```

Comment:
this should be a string, anything that is not a span with a bit of colors can bring some trouble with formattedMessages values